### PR TITLE
Feat:"added rating"

### PR DESCRIPTION
**Pull request by sadathshariff**
Added Rating
Rating

### CODE REVIEW

File: CSS/Components/rating.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+}
+
+.filled-star {
+  color: gold;
+}
+.rating-div {
+  border-radius: 5px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```
Comment (kjais1720) [inline CSS/Components/rating.css]: you can a use utility class for that

File: docs/rating/rating.html [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.4.0/styles/night-owl.min.css"
+      integrity="sha512-i5X6Fdn/ZqvGSqPrdMa3FgcpXM/Nr6YccSKFYT93zljl/HZDEpvBbE5Pxp91eiWGccZLrL/LDQJd7fjTRYsVaA=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="../../CSS/style.css" />
+  </head>
+  <body>
+    <div class="container">
```
Comment (kjais1720) [inline docs/rating/rating.html:21]: instead of a `div` this could be wrapped in the `header` tag

File: docs/rating/rating.html [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+            <li><a href="../inputs/input.html">Input</a></li>
+            <li><a href="../navbar/navbar.html">Navbar</a></li>
+            <li><a href="../lists/list.html">List</a></li>
+            <li><a href="../rating/rating.html">Rating</a></li>
+            <li><a href="../toast/toast.html">Toast</a></li>
+            <li><a href="../typography/typography.html">Typography</a></li>
+          </ul>
+        </div>
+        <!-- Main content -->
+        <div class="component-container">
```
Comment (kjais1720) [inline docs/rating/rating.html]: This `div` should be replaced by the `main` tag

File: docs/rating/rating.html [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+          </div>
+          <div class="example-code">
+            <pre>
+              <code class="html hljs xml">
+                <div class="rating">
+                  <i class="rating-icon fas fa-star filled-star"></i>
+                  <i class="rating-icon fas fa-star filled-star"></i>
+                  <i class="rating-icon fas fa-star filled-star"></i>
+                  <i class="rating-icon fas fa-star filled-star"></i>
+                  <i class="rating-icon fas fa-star filled-star"></i>
```
Comment (kjais1720) [inline docs/rating/rating.html]: the filled class could be used for icons other than the `star icons` as well (ex- the heart icon to add items to wishlist) , so give it a general name, like `filled-icon`

**Review by kjais1720 — CHANGES_REQUESTED** [review]

File: docs/rating/rating.html (hunk 4 above)
Comment (sadathshariff) [reply docs/rating/rating.html]: Have made the necessary changes

**Review by kjais1720 — APPROVED** [review]
All good now 👍

**Review by shubhamkumar648 — APPROVED** [review]
code looks good to merge

**Pull request merged**